src/main/java/address/sync/RemoteService.java

```diff
@@ -388,7 +388,7 @@ private long getRateResetFromHeader(HashMap<String, String> header) {
     private List<Person> convertToPersonList(List<CloudPerson> cloudPersonList) {
         List<Person> convertedList = new ArrayList<>();
         cloudPersonList.stream()
-                .forEach(CloudPerson -> convertedList.add(convertToPerson(CloudPerson)));
+                .forEach(cloudPerson -> convertedList.add(convertToPerson(cloudPerson)));
 
         return convertedList;
```

Comment:
:P

On Thu, Jul 7, 2016, 18:37 Martin Choo <notifications@github.com> wrote:

> In src/main/java/address/sync/RemoteService.java
> <https://github.com/HubTurbo/addressbook/pull/436#discussion_r69885406>:
>
> > @@ -388,7 +388,7 @@ private long getRateResetFromHeader(HashMap<String, String> header) {
> >      private List<Person> convertToPersonList(List<CloudPerson> cloudPersonList) {
> >          List<Person> convertedList = new ArrayList<>();
> >          cloudPersonList.stream()
> > -                .forEach(CloudPerson -> convertedList.add(convertToPerson(CloudPerson)));
> > +                .forEach(cloudPerson -> convertedList.add(convertToPerson(cloudPerson)));
> >
> >          return convertedList;
>
> Noted. map should be used here instead of filter tho 😛
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/HubTurbo/addressbook/pull/436/files/7d5e55d449b1f6e200c71e6cb87ac01e8666a7ab#r69885406>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AHjE2JzVuMAzEK2fFGt5bh7NjfSU8EaHks5qTNbsgaJpZM4JG164>
> .
>